ldclient/event_consumer.py

```diff
@@ -13,11 +13,10 @@
 
 
 class EventConsumerImpl(Thread, EventConsumer):
-    def __init__(self, event_queue, sdk_key, config):
+    def __init__(self, event_queue, config):
```

Comment:
This (and below) is a little harder to rationalize the backwards-compatible way of overlaying an optional `sdk_key` argument on the config.

Maybe doing that, and marking it `[deprecated](https://wiki.python.org/moin/PythonDecoratorLibrary#Generating_Deprecation_Warnings)` would be the way to go?  Then we can remove it in the next major release.